### DISCUSSION TITLE
Pomysł na dodatkową optymalizacje API batch offer modification

### DISCUSSION THREAD

**Discussion by bugproof** [allegro/allegro-api#9430] [Suggestions - resolved]
Myślę, że gdyby ten zasób pozwalał na przesłanie tablicy z kryteriami i zmianami tzn. zezwolił na modyfikacje różnych stanów i cen różnych ofert byłoby to spore ulepszenie.

np zamiast

```json
{
    "modification": {},
    "offerCriteria": []
}
```

to 

```json
[
	{
	    "modification": {},
	    "offerCriteria": []
	},
    {
	    "modification": {},
	    "offerCriteria": []
	},
]
```

**Comment by PrzemyslawLukanowski** [comment]
Jakiś czas temu wdrożyliśmy już [nowe zasoby do grupowej edycji ceny i stanu magazynowego
](https://developer.allegro.pl/news/zarzadzanie-ofertami-wdrozylismy-nowe-zasoby-do-grupowej-edycji-ceny-i-stanu-magazynowego-oAD1mwW0lHX) - zamykam sugestię.
